### DISCUSSION TITLE
Optimized parameters

### DISCUSSION THREAD

**Discussion by nikparastf** [kschan0214/sepia#19] [Q&A]
hi and with the best regards,
thanks for your amazing tools.
im working on SEPIA
there are different tools for different processing parts,such as MEDI,VSHARP,TKD and ....
but can u help me to find optimized parameters for each one of these ??
for example threshold for TKD   or lambda for MEDI or others...
thanks

**Comment by kschan0214** [comment]
Hi,

The optimised parameters depend on multiple factors such as data quality and the research question you have. A universal set of parameters that works for all data unfortunately does not exist and you have to exam it by yourself. If you are not very familiar with QSM, I suggest you starting from finding an optimised set of parameters on a more straightforward model (e.g. TKD or NDI). You can do so by testing a range of parameters and see which can produce the most naturally looking image. The default values are usually a good middle point of testing (e.g. testing values lower or higher than the default values). Once you obtain the best result you can achieve using this model, you can use it as a reference to see if you can achieve a better result using other more sophisticated models.

Hope this can be helpful and good luck!

Kwok
